.github/workflows/copilot_deploy.yml

```diff
@@ -207,57 +96,15 @@ jobs:
       environment: dev
```

Comment:
Why default to false for e2e?

Reply:
We'll discuss further when you're back - for now this is for consistency between all the repos pending deciding how we want to use dev/test.